meteor-bundle-main.js

```diff
@@ -14,8 +14,8 @@ var https = require('https');
 var net = require('net');
 var url = require('url');
 
-// Borrow the "node-forge" dependency of our meteor-node-forge package.
-var forge = require('./programs/server/npm/node_modules/meteor/meteor-node-forge/node_modules/node-forge');
+// Use the "node-forge" package from npm
```

Comment:
I'm the one who created this wrapper package, and I'm very happy to see it go away.